test/generateDefinition.test.ts

```diff
@@ -95,7 +95,7 @@ describe('GenerateDefinition', () => {
             ]},
             configureReporting: {
                 1: [
-                    ['msTemperatureMeasurement', [reportingItem('measuredValue', 10, repInterval.HOUR, 100)]],
+                    ['msTemperatureMeasurement', [reportingItem('measuredValue', 10, repInterval.HOUR, 1)]],
```

Comment:
This will make the sensor report changes of `0.01`C, I think thats too much.

Reply:
You are right, reverted this change.